setup.py

```diff
@@ -16,12 +16,12 @@
 
     python_requires=">=3.6",
     install_requires=[
-        "celery[redis]==4.4.0",
+        "celery[redis]==4.4.2",
         "chord_lib[flask]==0.7.0",
         "Flask>=1.1,<2.0",
         "requests>=2.23,<3.0",
         "requests-unixsocket>=0.2.0,<0.3.0",
-        "toil@git+https://github.com/afrosimon/toil@disable-content-length",
+        "toil==4.0.0"
```

Comment:
ideally this should have a trailing comma too still but nbd

Reply:
Hm you've got a PEP8 reference for this?